packages/components/tests/dummy/app/templates/components/form/file-input.hbs

```diff
@@ -0,0 +1,111 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "Form::FileInput Component"}}
+
+<Shw::Text::H1>Form::FileInput</Shw::Text::H1>
+
+<section data-test-percy>
+  <button id="dummy-toggle-highlight" type="button" {{on "click" this.toggleHighlight}}>
+    {{if this.showHighlight "Hide" "Show"}}
+    layout highlight
+  </button>
+
+  <Shw::Text::H2>“Base” control</Shw::Text::H2>
+
+  <div class="{{if this.showHighlight 'shw-component-form-file-input-layout-highlight'}}">
+    <Shw::Flex {{style gap="2rem"}} as |SF|>
+      <SF.Item @label="FileInput">
+        <Shw::Text::Body>
+          <code>Input[type=file]</code>
+          (styled via
+          <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/::file-selector-button">
+            <code>::file-selector-button</code>
+          </a>)
+        </Shw::Text::Body>
+        <Hds::Form::FileInput::Base />
+      </SF.Item>
+
+      <SF.Item @label="HdsButton for visual reference">
+        <Shw::Text::Body><code>Hds::Button[@color=secondary]</code></Shw::Text::Body>
+        <Hds::Button @icon="upload" @text="Choose file" @color="secondary" />
+      </SF.Item>
+    </Shw::Flex>
+
+    <Shw::Text::H2>States</Shw::Text::H2>
+
+    <Shw::Flex {{style gap="2rem"}} as |SF|>
+      {{#each @model.STATES as |state|}}
+        <SF.Item @label={{state}}>
+          <Hds::Form::FileInput::Base mock-state-value={{state}} />
+        </SF.Item>
+      {{/each}}
+      <SF.Item @label="disabled">
+        <Hds::Form::FileInput::Base mock-state-value="disabled" disabled />
+      </SF.Item>
+    </Shw::Flex>
+
+    <Shw::Divider @level={{2}} />
+
+    <Shw::Text::H3>HdsButton states for visual reference</Shw::Text::H3>
+    <Shw::Flex {{style gap="2rem"}} as |SF|>
+      {{#each @model.STATES as |state|}}
+        <SF.Item @label={{state}}>
+          <Hds::Button @icon="upload" @text="Choose file" @color="secondary" mock-state-value={{state}} />
+        </SF.Item>
+      {{/each}}
+      <SF.Item @label="disabled">
+        <Hds::Button @icon="upload" @text="Choose file" @color="secondary" mock-state-value="disabled" disabled />
+      </SF.Item>
+    </Shw::Flex>
```

Comment:
```suggestion
```

This was extremely confusing for me, I was changing the properties on the `FileInput` CSS and expecting to see them reflected here, only to understand that this is actually an `Hds::Button`.